### PR TITLE
ci: add a reminder to manually deploy merged changes

### PR DESCRIPTION
**Pull request by JacobCoffee**
## What

Adds a reminder comment from GH actions to manually deploy changes

### CODE REVIEW

File: .github/workflows/deployminder.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - name: Comment on PR
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: '@JacobCoffee Don\'t forget to pull the latest changes on `salt.nyc1.psf.io`!'
```
Comment (ewdurbin) [inline .github/workflows/deployminder.yml:24]: Heh, if you want the mention! I don't think it needs to be this targeted cause on your merges you would get an email notification of the comment regardless, right?

**Review by ewdurbin — APPROVED** [review]

File: .github/workflows/deployminder.yml (hunk 1 above)
Comment (JacobCoffee) [inline .github/workflows/deployminder.yml:24]:
```suggestion
              body: 'Don\'t forget to pull the latest changes on `salt.nyc1.psf.io`!'
```

**Comment by JacobCoffee** [comment]
DoesItWork™?

**Comment by JacobCoffee** [comment]
It do.

**Comment by JacobCoffee** [comment]
should probably scope the comment to only changed files in salt/ and pillar/ 😅

**Pull request merged**